skfda/ml/classification/_depth_classifiers.py

```diff
@@ -104,3 +108,120 @@ def predict(self, X):
         ]
 
         return self.classes_[np.argmax(depths, axis=0)]
+
+
+class DDTransform(BaseEstimator, TransformerMixin):
```

Comment:
Move this to dimensionality_reduction/feature_construction or something like this (we will discuss it).